diagnostic.md

```diff
@@ -45,7 +48,7 @@ while (x < 5) {
 
 How many times does the loop run?  What's the value of `x` after the while loop?
 
-_Replace this text with your answer._
+It runs 4 times and x is 8 after the while loop.
```

Comment:
The latter is correct, but the loop actually runs 3 times. Since x = 1 satisfies the loop condition (x < 5), the first iteration runs and x = 1 + 1 = 2.  x = 2 satisfies the loop condition, so it runs a second time; x = 2 + 2 = 4. x = 4 also satisfies the loop condition, so it runs a third time, x = 4 + 4 = 8. Now, x = 8 no longer satisfies the loop condition so the loop stops running.